common/djangoapps/pipeline_mako/tests/test_render.py

```diff
@@ -39,7 +39,7 @@ class RequireJSPathOverridesTest(TestCase):
     def test_requirejs_path_overrides(self):
         result = render_require_js_path_overrides(self.OVERRIDES)
         # To make the string comparision easy remove the whitespaces
-        self.assertEqual(list(map(str.strip, result.splitlines())), self.OVERRIDES_JS)
+        self.assertCountEqual(sorted(list(map(str.strip, result.splitlines()))), sorted(self.OVERRIDES_JS))
```

Comment:
Given that the assertion ignores the order of the compared sequences, do we need the `sorted()` calls?